common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1107,7 +1129,7 @@ def test_discussion_privileged_user_can_edit_team(self, role):
 
 @attr('shard_5')
 @ddt.ddt
-class TeamPageTest(TeamsTabBase):
+class TeamPageTest(TeamsTabBase, EventsTestMixin):
```

Comment:
This shouldn't need to inherit from `EventsTestMixin`, since `TeamsTabBase` already has it.